code/game/objects/items/theft_items.dm

```diff
@@ -202,13 +247,52 @@
 		if(ismob(loc))
 			to_chat(loc, "<span class='warning'>[src] is permanently sealed, [sliver] is safely contained.</span>")
 
+/obj/item/nuke_core_container/supermatter/proc/unload(obj/item/retractor/supermatter/I, mob/user)
+	if(!istype(I) || I.sliver)
+		return
+	sliver.forceMove(I)
+	I.sliver = sliver
+	sliver = null
+	I.icon_state = "supermatter_tongs"
+	I.item_state = "supermatter_tongs"
+	I.icon_state = "supermatter_tongs_loaded"
+	I.item_state = "supermatter_tongs_loaded"
+	icon_state = "core_container_cracked_empty"
+	to_chat(user, "<span class='notice'>You carefully pick up [I.sliver] with [I].</span>")
+
 /obj/item/nuke_core_container/supermatter/attackby(obj/item/retractor/supermatter/tongs, mob/user)
 	if(istype(tongs))
-		//try to load shard into core
-		load(tongs, user)
+		if(cracked)
+			//lets take that shard out
+			unload(tongs, user)
+		else
+			//try to load shard into core
+			load(tongs, user)
 	else
 		return ..()
 
+/obj/item/nuke_core_container/supermatter/attack_hand(mob/user)
+	if(cracked && sliver) //What did we say about touching the shard...
+		if(!isliving(user) || user.status_flags & GODMODE)
+			return FALSE
+		user.visible_message("<span class='danger'>[user] reaches out and tries to pick up [src]. [user.p_their()] body starts to glow and bursts into flames before flashing into dust!</span>",
+				"<span class='userdanger'>You reach for [src] with your hands. That was dumb.</span>",
+				"<span class='hear'>Everything suddenly goes silent.</span>")
```

Comment:
Sidenote. `hear` is not a css class
![image](https://user-images.githubusercontent.com/15887760/130231705-66f01899-2f87-40d7-9e80-8f361a905634.png)

Reply:
fixed